src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,23 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	int max_miners = g_maxMiners.Get();
+	if ( max_miners >= 0 && ( buildable == BA_H_DRILL || buildable == BA_A_LEECH ) )
+	{
+		int miners = 0;
+		ForEntities<MiningComponent> ( [&](Entity& entity, MiningComponent& mining )
```

Comment:
Delete the variable name `mining` here to avoid an unused variable warning.

LGTM

Reply:
I'm not sure how to do that...

Reply:
Literally delete the word `mining`